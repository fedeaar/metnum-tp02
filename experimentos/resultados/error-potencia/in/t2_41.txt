6.698000000000000000e+06 2.472328000000000000e+06 1.202334000000000000e+06 4.363320000000000000e+05 -4.917200000000000000e+05 -5.346340000000000000e+05 1.301175000000000000e+06 -9.896190000000000000e+05 3.687060000000000000e+05 2.042915000000000000e+06 -2.684684000000000000e+06 -1.723446000000000000e+06 5.185070000000000000e+05 -6.036450000000000000e+05 -8.451730000000000000e+05 -2.215300000000000000e+04 2.504810000000000000e+05 -1.591080000000000000e+05 -2.541822000000000000e+06 -1.598000000000000000e+06
2.472328000000000000e+06 4.745618000000000000e+06 8.361660000000000000e+05 1.303469000000000000e+06 3.304560000000000000e+05 -4.770640000000000000e+05 1.861694000000000000e+06 -7.746760000000000000e+05 -1.317647000000000000e+06 1.124812000000000000e+06 -1.049881000000000000e+06 -2.540370000000000000e+06 1.441083000000000000e+06 -1.851675000000000000e+06 -1.005316000000000000e+06 -1.033168000000000000e+06 7.211900000000000000e+04 3.474500000000000000e+05 -1.539323000000000000e+06 7.789600000000000000e+04
1.202334000000000000e+06 8.361660000000000000e+05 6.391409000000000000e+06 -2.143910000000000000e+05 -3.123385000000000000e+06 1.461550000000000000e+05 1.852295000000000000e+06 -5.950060000000000000e+05 1.029898000000000000e+06 -1.069278000000000000e+06 1.165460000000000000e+05 -1.113565000000000000e+06 4.922990000000000000e+05 -2.032774000000000000e+06 -7.418500000000000000e+04 4.175310000000000000e+05 1.070871000000000000e+06 -1.935430000000000000e+05 4.663210000000000000e+05 1.285174000000000000e+06
4.363320000000000000e+05 1.303469000000000000e+06 -2.143910000000000000e+05 6.205136000000000000e+06 1.081688000000000000e+06 -6.184020000000000000e+05 -3.678380000000000000e+05 3.090717000000000000e+06 4.116340000000000000e+05 1.201249000000000000e+06 -7.402700000000000000e+04 -3.097440000000000000e+05 -1.234080000000000000e+05 1.005649000000000000e+06 2.592320000000000000e+05 5.149820000000000000e+05 -4.681400000000000000e+04 1.156789000000000000e+06 1.851466000000000000e+06 -2.876360000000000000e+05
-4.917200000000000000e+05 3.304560000000000000e+05 -3.123385000000000000e+06 1.081688000000000000e+06 6.465091000000000000e+06 6.871910000000000000e+05 -2.286504000000000000e+06 7.032990000000000000e+05 6.681800000000000000e+04 6.010610000000000000e+05 -2.369094000000000000e+06 9.614870000000000000e+05 6.612870000000000000e+05 8.742020000000000000e+05 -2.390400000000000000e+05 -4.498080000000000000e+05 7.897520000000000000e+05 -1.450241000000000000e+06 1.836782000000000000e+06 -7.927570000000000000e+05
-5.346340000000000000e+05 -4.770640000000000000e+05 1.461550000000000000e+05 -6.184020000000000000e+05 6.871910000000000000e+05 6.085035000000000000e+06 -1.887977000000000000e+06 -4.427521000000000000e+06 2.053854000000000000e+06 -3.348506000000000000e+06 5.637800000000000000e+04 -6.567330000000000000e+05 -5.655760000000000000e+05 3.206760000000000000e+05 2.390076000000000000e+06 9.139920000000000000e+05 -1.881563000000000000e+06 2.405782000000000000e+06 2.544064000000000000e+06 6.029260000000000000e+05
1.301175000000000000e+06 1.861694000000000000e+06 1.852295000000000000e+06 -3.678380000000000000e+05 -2.286504000000000000e+06 -1.887977000000000000e+06 6.722451000000000000e+06 1.683880000000000000e+05 -1.673371000000000000e+06 -2.114440000000000000e+05 -9.575970000000000000e+05 -7.134900000000000000e+05 8.381200000000000000e+04 -1.474193000000000000e+06 -1.234074000000000000e+06 -8.215980000000000000e+05 -1.673380000000000000e+05 -4.729410000000000000e+05 -2.418593000000000000e+06 5.877960000000000000e+05
-9.896190000000000000e+05 -7.746760000000000000e+05 -5.950060000000000000e+05 3.090717000000000000e+06 7.032990000000000000e+05 -4.427521000000000000e+06 1.683880000000000000e+05 7.960430000000000000e+06 -9.326660000000000000e+05 2.628419000000000000e+06 -9.238890000000000000e+05 3.967170000000000000e+05 -1.207236000000000000e+06 4.589280000000000000e+05 -1.364467000000000000e+06 1.020912000000000000e+06 8.617240000000000000e+05 -2.363931000000000000e+06 -1.063269000000000000e+06 3.766560000000000000e+05
3.687060000000000000e+05 -1.317647000000000000e+06 1.029898000000000000e+06 4.116340000000000000e+05 6.681800000000000000e+04 2.053854000000000000e+06 -1.673371000000000000e+06 -9.326660000000000000e+05 8.447855000000000000e+06 -4.088000000000000000e+03 -3.156570000000000000e+05 2.040175000000000000e+06 -1.269077000000000000e+06 2.027210000000000000e+05 -5.799290000000000000e+05 3.084624000000000000e+06 -1.101973000000000000e+06 -1.663448000000000000e+06 2.147370000000000000e+05 -3.124930000000000000e+05
2.042915000000000000e+06 1.124812000000000000e+06 -1.069278000000000000e+06 1.201249000000000000e+06 6.010610000000000000e+05 -3.348506000000000000e+06 -2.114440000000000000e+05 2.628419000000000000e+06 -4.088000000000000000e+03 8.309107000000000000e+06 -1.730706000000000000e+06 -1.048724000000000000e+06 7.630050000000000000e+05 -6.602450000000000000e+05 -1.099505000000000000e+06 1.190721000000000000e+06 1.870290000000000000e+05 -2.997107000000000000e+06 -1.825000000000000000e+03 -2.395280000000000000e+05
-2.684684000000000000e+06 -1.049881000000000000e+06 1.165460000000000000e+05 -7.402700000000000000e+04 -2.369094000000000000e+06 5.637800000000000000e+04 -9.575970000000000000e+05 -9.238890000000000000e+05 -3.156570000000000000e+05 -1.730706000000000000e+06 6.217510000000000000e+06 3.839550000000000000e+05 -1.270563000000000000e+06 2.024522000000000000e+06 1.312105000000000000e+06 -2.185841000000000000e+06 1.751027000000000000e+06 1.655778000000000000e+06 4.199910000000000000e+05 -1.087535000000000000e+06
-1.723446000000000000e+06 -2.540370000000000000e+06 -1.113565000000000000e+06 -3.097440000000000000e+05 9.614870000000000000e+05 -6.567330000000000000e+05 -7.134900000000000000e+05 3.967170000000000000e+05 2.040175000000000000e+06 -1.048724000000000000e+06 3.839550000000000000e+05 7.343021000000000000e+06 9.226480000000000000e+05 2.683601000000000000e+06 -1.585465000000000000e+06 1.152083000000000000e+06 2.068133000000000000e+06 -1.850500000000000000e+05 -6.687700000000000000e+05 -4.464280000000000000e+05
5.185070000000000000e+05 1.441083000000000000e+06 4.922990000000000000e+05 -1.234080000000000000e+05 6.612870000000000000e+05 -5.655760000000000000e+05 8.381200000000000000e+04 -1.207236000000000000e+06 -1.269077000000000000e+06 7.630050000000000000e+05 -1.270563000000000000e+06 9.226480000000000000e+05 6.785786000000000000e+06 -9.922090000000000000e+05 4.413580000000000000e+05 8.297180000000000000e+05 6.673160000000000000e+05 3.484166000000000000e+06 -2.232300000000000000e+04 -1.276399000000000000e+06
-6.036450000000000000e+05 -1.851675000000000000e+06 -2.032774000000000000e+06 1.005649000000000000e+06 8.742020000000000000e+05 3.206760000000000000e+05 -1.474193000000000000e+06 4.589280000000000000e+05 2.027210000000000000e+05 -6.602450000000000000e+05 2.024522000000000000e+06 2.683601000000000000e+06 -9.922090000000000000e+05 4.693369000000000000e+06 7.136510000000000000e+05 -5.048960000000000000e+05 7.152110000000000000e+05 7.790010000000000000e+05 7.036250000000000000e+05 -3.241670000000000000e+05
-8.451730000000000000e+05 -1.005316000000000000e+06 -7.418500000000000000e+04 2.592320000000000000e+05 -2.390400000000000000e+05 2.390076000000000000e+06 -1.234074000000000000e+06 -1.364467000000000000e+06 -5.799290000000000000e+05 -1.099505000000000000e+06 1.312105000000000000e+06 -1.585465000000000000e+06 4.413580000000000000e+05 7.136510000000000000e+05 5.768532000000000000e+06 -2.411580000000000000e+05 -1.422684000000000000e+06 1.479944000000000000e+06 3.423168000000000000e+06 1.878390000000000000e+05
-2.215300000000000000e+04 -1.033168000000000000e+06 4.175310000000000000e+05 5.149820000000000000e+05 -4.498080000000000000e+05 9.139920000000000000e+05 -8.215980000000000000e+05 1.020912000000000000e+06 3.084624000000000000e+06 1.190721000000000000e+06 -2.185841000000000000e+06 1.152083000000000000e+06 8.297180000000000000e+05 -5.048960000000000000e+05 -2.411580000000000000e+05 6.128926000000000000e+06 -2.035629000000000000e+06 1.059950000000000000e+06 1.484690000000000000e+05 -9.193100000000000000e+04
2.504810000000000000e+05 7.211900000000000000e+04 1.070871000000000000e+06 -4.681400000000000000e+04 7.897520000000000000e+05 -1.881563000000000000e+06 -1.673380000000000000e+05 8.617240000000000000e+05 -1.101973000000000000e+06 1.870290000000000000e+05 1.751027000000000000e+06 2.068133000000000000e+06 6.673160000000000000e+05 7.152110000000000000e+05 -1.422684000000000000e+06 -2.035629000000000000e+06 7.992427000000000000e+06 4.148030000000000000e+05 1.522013000000000000e+06 -2.646480000000000000e+05
-1.591080000000000000e+05 3.474500000000000000e+05 -1.935430000000000000e+05 1.156789000000000000e+06 -1.450241000000000000e+06 2.405782000000000000e+06 -4.729410000000000000e+05 -2.363931000000000000e+06 -1.663448000000000000e+06 -2.997107000000000000e+06 1.655778000000000000e+06 -1.850500000000000000e+05 3.484166000000000000e+06 7.790010000000000000e+05 1.479944000000000000e+06 1.059950000000000000e+06 4.148030000000000000e+05 7.386146000000000000e+06 9.936120000000000000e+05 -9.912280000000000000e+05
-2.541822000000000000e+06 -1.539323000000000000e+06 4.663210000000000000e+05 1.851466000000000000e+06 1.836782000000000000e+06 2.544064000000000000e+06 -2.418593000000000000e+06 -1.063269000000000000e+06 2.147370000000000000e+05 -1.825000000000000000e+03 4.199910000000000000e+05 -6.687700000000000000e+05 -2.232300000000000000e+04 7.036250000000000000e+05 3.423168000000000000e+06 1.484690000000000000e+05 1.522013000000000000e+06 9.936120000000000000e+05 8.547488000000000000e+06 1.379938000000000000e+06
-1.598000000000000000e+06 7.789600000000000000e+04 1.285174000000000000e+06 -2.876360000000000000e+05 -7.927570000000000000e+05 6.029260000000000000e+05 5.877960000000000000e+05 3.766560000000000000e+05 -3.124930000000000000e+05 -2.395280000000000000e+05 -1.087535000000000000e+06 -4.464280000000000000e+05 -1.276399000000000000e+06 -3.241670000000000000e+05 1.878390000000000000e+05 -9.193100000000000000e+04 -2.646480000000000000e+05 -9.912280000000000000e+05 1.379938000000000000e+06 7.441950000000000000e+06
