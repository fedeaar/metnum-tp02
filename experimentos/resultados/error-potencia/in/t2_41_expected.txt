2.086271683495309576e+07
1.699184164509795979e+07
1.435855893345644698e+07
1.308755614503486082e+07
1.230878511271550506e+07
1.011474382315306365e+07
9.218538353680003434e+06
8.395877124547777697e+06
6.888572184509735554e+06
5.132696577800220810e+06
4.554678955765925348e+06
4.196298781785723753e+06
3.398083875349756330e+06
2.225471830988082569e+06
1.659996930418846197e+06
1.387147643247045809e+06
9.004549193023998523e+05
5.028061772116365610e+05
1.438883006688725727e+05
6.572850313231814653e+03
